frb_codegen/src/library/codegen/config/internal_config_parser.rs

```diff
@@ -360,4 +366,48 @@ mod tests {
 
         Ok(())
     }
+
+    #[test]
+    #[serial]
+    fn test_compute_path_map() -> anyhow::Result<()> {
+        let result = super::compute_path_map(&PathBuf::from("src/api/api.rs"))?;
+        assert_eq!(result.common, PathBuf::from("src/api/api.rs"));
+        assert_eq!(result.io, PathBuf::from("src/api/api.io.rs"));
+        assert_eq!(result.web, PathBuf::from("src/api/api.web.rs"));
+        Ok(())
+    }
+    #[test]
+    #[serial]
+    fn test_compute_path_map_with_glob() -> anyhow::Result<()> {
+        let result = super::compute_path_map(&PathBuf::from("src/api/*.rs"))?;
+        assert_eq!(result.common, PathBuf::from("src/api/*.rs"));
+        assert_eq!(result.io, PathBuf::from("src/api/*.io.rs"));
+        assert_eq!(result.web, PathBuf::from("src/api/*.web.rs"));
+        Ok(())
+    }
+    #[test]
+    #[serial]
+    fn test_compute_path_map_faulty() -> anyhow::Result<()> {
+        let result = super::compute_path_map(&PathBuf::from("src/api"));
+        assert!(result.is_err());
+        assert!(result
+            .err()
+            .unwrap()
+            .to_string()
+            .contains("Cannot use the path configuration"));
+        Ok(())
+    }
+
+    #[test]
+    #[serial]
+    fn test_parse_rust_output_faulty() -> anyhow::Result<()> {
+        let result = body("library/codegen/config/internal_config_parser/faulty_rust_output");
+        assert!(result.is_err());
+        assert!(result
+            .err()
+            .unwrap()
+            .to_string()
+            .contains("rust_output: is wrong:"));
```

Comment:
nit: sth like

```suggestion
        let e = result.err().unwrap().to_string();
        assert!(e.contains("rust_output: is wrong:"));
        assert!(e.contains("A path for input/output needs to include the file name"));
```

to ensure we do have the new friendly error message :)

Reply:
Good suggestion!